### DISCUSSION TITLE
How to create a wrap table surrounded by text

### DISCUSSION THREAD

**Discussion by Boltzmachine** [typst/typst#4756] [Q&A]
I found [wrap-it](https://typst.app/universe/package/wrap-it/) but this does not work for tables.

![46zkN](https://github.com/user-attachments/assets/416e2b0f-1d96-43e4-839e-7f35075ebd80)

**Comment by emilyyyylime** [comment]
That image looks fine to me. Which part is 'not working'?

**Comment by ErrWare** [comment]
I'm having this issue as well. I tried following the example in the [wrap-it documentation](https://typst.app/universe/package/wrap-it), but it did not work when I used a table instead of a figure.

Their example:
```typ
#wrap-content(
fig,
body,
align: bottom + right,
column-gutter: 2em
)
```

My code:
```typ
#import "@preview/wrap-it:0.1.1": wrap-content
#import "@preview/zero:0.6.1": num, format-table, zi
#let doX = {
    show: format-table(none, auto)
    table(columns:(auto,auto),
    align: (left, right),
    table.cell(colspan: 2, align:center)[Time to do X],
    [A], [#zi.minute[10]],
    [A], [#zi.minute[1]],
    [A], [#zi.s[10]],
    [A], [#zi.s[1]],
    [A], [#zi.s[0]],)}
#let body = par(justify:true, lorem(100))
```

My result:
<img width="1044" height="525" alt="Screenshot from 2026-02-16 15-27-37" src="https://github.com/user-attachments/assets/09df88e0-36f1-4aa1-ba66-7fdc523b7585" />
